cmd/mockrecord.go

```diff
@@ -65,22 +65,27 @@ func (mr *MockRecord) GetCmd() *cobra.Command {
 				return
 			}
 
-
+			disableTele, err := cmd.Flags().GetBool("disableTele")
```

Comment:
Please rename this variable to enableTelemetry and default value should be true. This is what we used in v1. It is just because people who don't know telemetry will think that it is something useful by looking at the name enableTelemetry and will not disable it anytime.